### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by drabiter**
syalalala.

### CODE REVIEW

**Comment by andesyudanto** [comment]
test comment..

File: README.md [hunk 1]
```diff
@@ -1,2 +1,4 @@
 # kodok
 a kodok repository
+
+1st test
```
Comment (drabiter) [inline README.md]: this changes is absurd.

**Comment by andesyudanto** [comment]
:ok_hand:

**Comment by drabiter** [comment]
boom

**Comment by drabiter** [comment]
booom

**Comment by drabiter** [comment]
boom

**Comment by drabiter** [comment]
:ok_hand:

**Comment by drabiter** [comment]
:ok_hand:

**Comment by drabiter** [comment]
:ok_hand:

**Comment by drabiter** [comment]
:ok_hand:

**Comment by drabiter** [comment]
:ok_hand:

**Comment by drabiter** [comment]
:ok_hand:

**Comment by drabiter** [comment]
:ok_hand:

**Comment by drabiter** [comment]
:ok_hand:

**Comment by drabiter** [comment]
:ok_hand:

**Comment by drabiter** [comment]
:ok_hand:

**Comment by drabiter** [comment]
:ok_hand:

**Comment by drabiter** [comment]
:ok_hand:

**Comment by drabiter** [comment]
:ok_hand:

**Comment by drabiter** [comment]
:ok_hand:

**Comment by drabiter** [comment]
:ok_hand:

**Comment by drabiter** [comment]
:ok_hand:

**Comment by drabiter** [comment]
:ok_hand:

**Comment by drabiter** [comment]
:ok_hand:

**Comment by drabiter** [comment]
:ok_hand:

**Comment by drabiter** [comment]
:ok_hand:

**Comment by drabiter** [comment]
:ok_hand:

**Comment by drabiter** [comment]
:ok_hand:

**Comment by gde-vt** [comment]
:ok_hand:

**Comment by gde-vt** [comment]
:ok_hand:

**Comment by gde-vt** [comment]
:ok_hand:

**Comment by gde-vt** [comment]
:ok_hand:

**Comment by gde-vt** [comment]
:ok_hand:

**Comment by gde-vt** [comment]
:ok_hand:

**Comment by gde-vt** [comment]
:ok_hand:

**Comment by gde-vt** [comment]
:ok_hand:

**Comment by gde-vt** [comment]
:ok_hand:

**Comment by gde-vt** [comment]
:ok_hand:

**Comment by gde-vt** [comment]
:ok_hand:

**Comment by gde-vt** [comment]
:ok_hand:

**Comment by gde-vt** [comment]
:ok_hand:

**Comment by gde-vt** [comment]
:ok_hand:

**Comment by gde-vt** [comment]
:ok_hand:

**Comment by andesyudanto** [comment]
#design

**Pull request merged**